quotefault/__init__.py

```diff
@@ -216,11 +217,17 @@ def get():
     quotes = get_quote_query(speaker = request.args.get('speaker'),
         submitter = request.args.get('submitter')).limit(20).all()
 
+    adjust = []
+    for obj in quotes:
+        quote = obj[0]
+        quote.quote_time = quote.quote_time.astimezone(pytz.timezone('America/New_York'))
+        adjust.append((quote,obj[1]))
+

```

Comment:
Can't you just `obj[0].quote_time=quote.quote_time.astimezone(pytz.timezone('America/New_York'))` instead of creating a new list?

Reply:
You can delete likes 220,224 and go back to using quotes instead of adjust